### DISCUSSION TITLE
[iOS] How to run Xamarin Forms app in UIWindowScene?

### DISCUSSION THREAD

**Discussion by cncb-gh** [xamarin/Xamarin.Forms#14980] [Q&A] [answered]
Is there a way to run a Xamarin Forms app inside a UIWindowScene?  I tried the below WillConnect() function for UIWindowSceneDelegate but I get this when trying to create the new UIWindow: 
_Exception of type 'Foundation.ModelNotImplementedException' was thrown._

```
public override void WillConnect(UIScene scene, UISceneSession session, UISceneConnectionOptions connectionOptions)
{
        Xamarin.Forms.Forms.Init();
        UIWindowScene windowScene = (UIWindowScene)scene;
        this.Window = new UIWindow(windowScene);
}

```

**Comment by cncb-gh** [comment, marked as answer]
Turns out you need to implement the Window property like so:

_public override UIWindow Window { get; set; }_

**Reply by zurtox** [reply]
Did you have a raw example how did you do it? I'm having issues with the info.plist

**Reply by cncb-gh** [reply]
Sorry, I ended up not using Scenes because it messed up other things in my app.
